README.rst

```diff
@@ -32,10 +32,23 @@ Install it like this:
 Usage
 =====
 
+Python API
+----------
+The rest of the package consists of scripts that are meant to be invoked from the command line.
```

Comment:
This sentence doesn't make sense to me here since this is the "Python API" section. I think we can remove it!